www/apps/book/app/learn/fundamentals/modules/db-operations/page.mdx

```diff
@@ -683,3 +683,146 @@ class BlogModuleService {
   }
 }
 ```
+
+Here are examples of typical operations using a repository:
+
+```ts
+// Create a new blog post
+const post = await postRepository.create({
+  title: "Introduction to Medusa",
+  content: "Medusa is a composable commerce platform...",
+  author_id: "author_123",
+  status: "draft"
+})
+
+// Find a specific post
+const post = await postRepository.findOne({
+  where: { id: "post_123456" }
+})
```

Comment:
The `findOne` method isn't available on a data model repository so remove it please.